source/braille.py

```diff
@@ -1147,6 +1156,10 @@ def _getControlFieldForReportStart(
 	level = field.get("level")
 	if level:
 		props["positionInfo"] = {"level": level}
+	if role == controlTypes.Role.LIST:
+		childControlCount = field.get("_childcontrolcount")
+		if childControlCount:
+			props["positionInfo"] = {"childControlCount": childControlCount}
```

Comment:
Maybe slightly cleaner, though I don't mind either way:
```suggestion
	if (
		role == controlTypes.Role.LIST
		and (childControlCount := field.get("_childcontrolcount", 0)) > 0
	):
		props["positionInfo"] = {"childControlCount": childControlCount}
```